docs/topics/proc_using-hystrix-dashboard-to-monitor-the-circuit-breaker.adoc

```diff
@@ -1,5 +1,5 @@
 [id='using-hystrix-dashboard-to-monitor-the-circuit-breaker_{context}']
-= Using Hystrix Dashboard to Monitor the Circuit Breaker
+= Using hystrix dashboard to monitor the circuit breaker
```

Comment:
Hystrix is a name, needs to be "Hystrix".

Reply:
Actually I think "Hystrix Dashboard" is the entire name.

Reply:
The [wiki page](https://github.com/Netflix-Skunkworks/hystrix-dashboard/wiki) of the Hystrix Dashboard Github project also capitalizes both words in the name. So I'd say you're right, @lthon:)

Reply:
Of course, fixed including Dashboard, thanks.